sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -337,12 +338,12 @@ class RocksDBSuite extends SparkFunSuite {
     }
   }
 
-  ignore("ensure that concurrent update and cleanup consistent versions") {
+  test("ensure that concurrent update and cleanup consistent versions") {
     quietly {
       val numThreads = 20
       val numUpdatesInEachThread = 20
       val remoteDir = Utils.createTempDir().toString
-      @volatile var exception: Exception = null
+      @volatile val exception: ArrayBuffer[Exception] = _
```

Comment:
Name it with `exceptions`?

Reply:
Thanks, just temporarily adding more logs to this WIP PR for debugging the flaky test. Since the failure can't reproduce locally.